diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5d9c885..6146f6f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,16 +117,13 @@ INCLUDE_DIRECTORIES(${VTK_INCLUDE_DIR})
 
 # Qt
 set(CMAKE_AUTOMOC ON)
-if(${VTK_VERSION} VERSION_GREATER "6" AND VTK_QT_VERSION VERSION_GREATER "4")
+if(${VTK_VERSION} VERSION_GREATER "6")
     find_package(Qt5Widgets REQUIRED)
     include_directories(${Qt5Widgets_INCLUDE_DIRS})
     find_package(Qt5Xml REQUIRED)
     include_directories(${Qt5Xml_INCLUDE_DIRS})
     INCLUDE_DIRECTORIES(${Qt5_INCLUDE_DIRS})
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${Qt5Widgets_EXECUTABLE_COMPILE_FLAGS}")
-else()
-    FIND_PACKAGE(Qt4 REQUIRED QtCore QtGui QtXml)
-    INCLUDE( ${QT_USE_FILE} )
 endif()
 
 message(STATUS "Found package VTK. Using version " ${VTK_VERSION})
